bigbluebutton-tests/playwright/user/multiusers.js

```diff
@@ -267,6 +267,37 @@ class MultiUsers {
     await this.modPage.hasElement(e.multiUsersWhiteboardOn);
   }
 
+  async removeUser() {
+    await this.modPage.waitAndClick(e.userListItem);
+    await this.modPage.waitAndClick(e.removeUser);
+    await this.modPage.waitAndClick(e.removeUserConfirmationBtn);
+
```

Comment:
it will be good to add here an assertion to confirm the user has been removed from the user list (and in the test below as well)